test/unitTests/Tracking.js

```diff
@@ -0,0 +1,173 @@
+/* eslint-env mocha */
+/* eslint no-global-assign: 0 */
+var chai = require("chai");
+var expect = chai.expect;
+var sinon = require("sinon");
+var tracking = require("../../lib/events/tracking.js");
+var domUtils = require("../../lib/utils/dom.js");
+var assert = require("assert");
+
+
+
+describe("Unit Tests", function() {
```

Comment:
Can we call this "Tracking module unit tests"? Otherwise both blocks in Tracking.js and MenuFunction.js have the same name